comixed-app/src/main/java/org/comixedproject/ComiXedApp.java

```diff
@@ -52,7 +52,6 @@ public void onDestroy() {
     log.info("Closing application context");
     ((ConfigurableApplicationContext) this.applicationContext).close();
     log.info("Shutting down");
-    System.exit(0);
```

Comment:
Does simply closing the application context shutdown the application, or do we need to take other steps as well?

Reply:
Yes, it should. That's the documentation said.
I also looked into the threads and the they were looked because the system.exit was waiting for Spring exit.

Reply:
Okay.